index.html

```diff
@@ -273,7 +285,45 @@
 Privacy therefore depends on having an equivalently strong countervailing force.
 
 
-## Exclusion
+### Age Verification {#age}
+
+A number of governments have chosen to mandate the use of age verification
+for access to certain online services.
+There are many issues that might be discussed on the topic,
+including whether age verification is even appropriate [[SUPPRESS]],
+all beyond the scope of this document.
+An upcoming workshop will address the issue
+of age-based restrictions on content and services
+in detail [[AGE-WS]].
+
+How different jurisdictions have set out requirements for age verification vary,
+but a common theme is to place the responsibility on the service provider.
+The presentation of government-issued credentials
+is often recognized as an acceptable age verification method;
+in other cases, government credentials are one of a set of acceptable methods.
+
+Even if privacy concerns around surveillance are mitigated,
+the use of credentials in this setting creates another way in which
+people might become accustomed to providing their identity documents.
+Use of credentials for age verification therefore leads to credential overuse.
+
+The presentation of identity documents
+to access content that might be sensitive
+creates serious privacy issues.
+A great deal then depends on how credentials
+are requested, authorized ([[[#auth-use]]]), and handled.
+The selective disclosure methods
+(such as [SD-JWT](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt))
+that are used in some designs
+create an unacceptable outcome where governments and websites
+can collude to identify and track site visitors.
+The use of zero-knowledge proofs
+might prevent tracking in the presence of collusion,
+though it might not address other issues,
+such as the potential for exclusion ([[[#exclusion]]]).
```

Comment:
I realized that this paragraph is saying almost the same thing as #30, and might not be specific to age verification. Would it make sense to move it there?

Reply:
Yeah, I might drop this paragraph here and work on #30, which only need to note that SD-whatever is linkable.